### DISCUSSION TITLE
Disabling unsolicited messaging.

### DISCUSSION THREAD

**Discussion by mattheweby** [stepfunc/dnp3#351] [Q&A]
Our current DNP3 implementation doesn't use unsolicited messaging. We use a master with a high frequency scan for Classes 1,2,3 along with a lower periodicity integrity (Class 1,2,3,0) scan. I'm not seeing any documentation regarding the configuration of unsolicited messaging in an outstation application.

**Comment by jadamcrain** [comment]
Hi @mattheweby. 

In C#?

Set `OutstationConfig.Features.Unsolicited = false`:

https://docs.stepfunc.io/dnp3/1.5.2/dotnet/classdnp3_1_1OutstationFeatures.html
